src/services/tag-service.js

```diff
@@ -0,0 +1,11 @@
+import { StorageService } from "./storage";
+
+export class TagListService {
+  get() {
```

Comment:
get and set should be static methods -- so you won't need to instantiate this class